### ISSUE TITLE
Display full whisper recipient details on received messages, even if you are the only recipient of the message.

### ISSUE THREAD

**Issue by aaclayton** [foundryvtt/foundryvtt#6469]
Originally in GitLab by @cs96and

### Feature Summary

Currently, if you receive a message whispered only to yourself, the recipient information is not displayed.  The sender does see that information though.  E.g. here is what the sender sees...

![image](https://gitlab.com/foundrynet/foundryvtt/uploads/39699ec263b98bd23ea15d7f88f2ce0c/image.png)

And this is what the receiver sees...

![image](https://gitlab.com/foundrynet/foundryvtt/uploads/6c1249403fd14965dbd85c34a63e0816/image.png)

You can see the "To:" information is missing from the second message, but is present on the third one, which has been sent to multiple people.

### User Experience

The receiver should see the whisper recipient on all received whispers.  Relying on just the background colour to determine if the message is a whisper or not is not good UX, and may be difficult for people with colour blindness.  It also feels inconsistent with the messages that are whispered to multiple people, which do show the whisper recipients.

### Priority/Importance

Not massively important, but it feels easy enough to fix to squeeze into the next v9 release :smile:

**Comment by cs96and** [comment]
This bug is still present in v10 dev 2.  Can we get it fixed please?

**Comment by cs96and** [comment]
I took a quick look and I think the problem is in `ChatMessage.getHTML()` when it constructs the message data.

```javascript
    // Construct message data
    const messageData = {
      message: data,
      user: game.user,
      author: this.user,
      alias: this.alias,
      cssClass: [
        this.type === CONST.CHAT_MESSAGE_TYPES.IC ? "ic" : null,
        this.type === CONST.CHAT_MESSAGE_TYPES.EMOTE ? "emote" : null,
        isWhisper ? "whisper" : null,
        this.blind ? "blind": null
      ].filterJoin(" "),
      isWhisper: this.whisper.some(id => id !== game.user.id),
      canDelete: game.user.isGM,  // Only GM users are allowed to have the trash-bin icon in the chat log itself
      whisperTo: this.whisper.map(u => {
        let user = game.users.get(u);
        return user ? user.name : null;
      }).filterJoin(", ")
    };
```

You can see that `isWhisper` is only set to true if there are some other whisper recipients apart from yourself.  Simply changeing this line to `isWhisper,` fixes the issue.

**Comment by aaclayton** [comment]
This was an intentional design, which is why it had not been prioritized for a fix - but I will review the situation again and either make the change or close it as wontfix. I think the accessibility argument is perhaps a persuasive one.
